resource-definitions/terraform-driver/backends/s3/s3-backend.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: entity.humanitec.io/v1b1
```

Comment:
Suggesting to add a small description to help the AI position the content (recommended practice by Kapa)
```suggestion
# This Resource Definition provisions an Amazon S3 bucket using another S3 bucket as the Terraform backend
apiVersion: entity.humanitec.io/v1b1
```